assets/js/pages/Profile/ProfilePage.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { toast } from 'react-hot-toast';
+
+import PageHeader from '@common/PageHeader';
+import ProfileForm from '@pages/Profile/ProfileForm';
+import { getUserProfile, updateUserProfile } from '@lib/api/users';
+
+function ProfilePage() {
+  const [errorsState, setErrors] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [saving, setSaving] = useState(false);
+  const [userState, setUser] = useState(null);
+
+  useEffect(() => {
+    getUserProfile()
+      .then(({ data: user }) => {
+        setUser(user);
+      })
+      .catch(() => {})
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
+
+  const updateProfile = (payload) => {
+    setSaving(true);
+    updateUserProfile(payload)
+      .then(({ data: updatedUser }) => {
+        toast.success('Profile changes saved!');
+        setUser(updatedUser);
+      })
+      .catch(
+        ({
+          response: {
+            data: { errors },
+          },
+        }) => {
+          setErrors(errors);
+        }
+      )
+      .finally(() => {
+        setSaving(false);
+      });
+  };
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+  const { fullname, email, username, abilities, id } = userState;
+  const isDefaultAdmin = id === 1;
```

Comment:
We have a `isAdmin` function in `lib/model/users.js` for his

Reply:
done!